rest/reference/exchanges_test.go

```diff
@@ -0,0 +1,52 @@
+package reference_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	polygon "github.com/polygon-io/client-go/rest"
+	"github.com/polygon-io/client-go/rest/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListExchanges(t *testing.T) {
+	c := polygon.New("API_KEY")
+
+	httpmock.ActivateNonDefault(c.Reference.HTTP.GetClient())
+	defer httpmock.DeactivateAndReset()
+
+	exchange1 := `{
```

Comment:
might be better to just put this directly in `expectedResponse` since you don't need to verify it by itself anymore